website/components/remote-plugin-docs/utils/fetch-github-file.js

```diff
@@ -0,0 +1,91 @@
+const fetch = require('isomorphic-unfetch')
+
+const GITHUB_API_TOKEN = process.env.GITHUB_API_TOKEN
+
+async function githubQuery(body, token) {
+  const result = await fetch('https://api.github.com/graphql', {
+    method: 'POST',
+    headers: {
+      Authorization: `bearer ${token}`,
+      ContentType: 'application/json',
+    },
+    body: JSON.stringify(body),
+  })
+  return await result.json()
+}
+
+//  Fetch a file from GitHub using the GraphQL API
+async function getGithubFile({ repo, branch, filePath }) {
+  const [repo_owner, repo_name] = repo.split('/')
+  //  Set up the GraphQL query
+  // (would be kinda nice to have this in a separate file, if possible)
```

Comment:
you should be able to load it from extension `.graphql`, i think nextjs-scripts injects a plaintext loader at this extension by default

Reply:
Ah yeah, makes sense! I think I did this when developing the scripts, just so I could easily run it. But I just realized that we might also need to use this outside NextJS when indexing content, so going to leave it inline for now till we can resolve the indexing challenge 🤖